### DISCUSSION TITLE
X-CIGALE fitting: Question about how to change E_cut

### DISCUSSION THREAD

**Discussion by HikaruF9** [mboquien/cigale#6] [Q&A] [answered]
Hi, I've used (X-)CIGAL, and I have one question on the parameter set-up of the X-CIGALE code.
I have tried the SED fitting using X-CIGALE to one source, which already has a detailed X-ray spectral result, which shows E_cut=130 keV.
Looks like the current X-CIGALE code has a fixed E_cut value of E_cut=300 keV, but is there a way to change the value?

I have quickly checked the total energy budget and it changes roughly 25% between E_cut = 130 keV and =300 keV at the energy range of E=14-195 keV, which might affect the total energy budget in the SED.

**Comment by mboquien** [comment]
@Guang91 This question is for you.

**Comment by Guang91** [comment, marked as answer]
That's a good point. I modify the code so that E_cut is a free parameter now. Could you install the "develop" version following the instructions (https://github.com/mboquien/cigale/discussions/2)? Please let me know if you find any issues (I've done some quick tests and it seems okay). 

BTW, X-CIGALE becomes a historical thing. Now, we call the new version as CIGALE v2022 (which includes all functions in the old X-CIGALE). see https://arxiv.org/abs/2201.03718 and https://cigale.lam.fr/news/

**Comment by HikaruF9** [comment]
Thank you for introducing the new great version! 
I've done fitting and it is working well, but I have one question. Is there any way to check the value of E_cut selected as the best fit model?

**Reply by mboquien** [reply]
You can find it under the `best.xray.E_cut` column. For the Bayesian version, it is under `bayes.xray.E_cut`.

**Reply by HikaruF9** [reply]
Thank you for your quick reply. I am using a wrong version. 

I didn’t switch to the develop branch. 
I will do "git checkout develop"

**Comment by Guang91** [comment]
okay, I see. Should use develop version.

**Reply by HikaruF9** [reply]
I can now select E_cut. Thank you so much!
